cirq/ops/matrix_gates_test.py

```diff
@@ -135,3 +136,77 @@ def test_two_qubit_extrapolate():
     assert cz2.extrapolate_effect(0).approx_eq(i)
     assert cz4.extrapolate_effect(0).approx_eq(i)
     assert cz2.extrapolate_effect(0.5).approx_eq(cz4)
+
+
+def test_single_qubit_diagram():
+    a = cirq.NamedQubit('a')
+    b = cirq.NamedQubit('b')
+    m = np.array([[1, 1j], [1j, 1]]) * np.sqrt(0.5)
+    c = cirq.Circuit.from_ops(
+        cirq.SingleQubitMatrixGate(m).on(a),
+        cirq.CZ(a, b))
+    assert re.match("""
+a: ───┌[             ]+┐───@───
+      │[0-9#\\.+\\-j ]+│   │
+      │[0-9#\\.+\\-j ]+│   │
+      └[             ]+┘   │
+       [             ]+    │
+b: ────[─────────────]+────@───
```

Comment:
Numpy array repr strings can have '#' in them?

Reply:
Removed.